app/controllers/slides_controller.rb

```diff
@@ -72,18 +71,19 @@ def create
     @slide.user_id = current_user.id
     if @slide.save
       slide = Slide.where('slides.key = ?', key).first
-      send_message({ id: slide.id, key: key }.to_json)
+      CloudConfig::SERVICE.send_message({ id: slide.id, key: key }.to_json)
       redirect_to slide_path(slide.id)
     else
-      render :new
+      render "slides/#{CloudConfig::service_name}/new"
```

Comment:
__[rubocop]__
* Style/ColonMethodCall: Do not use `::` for method calls. [:ref](https://github.com/bbatsov/ruby-style-guide#double-colons)